sql/core/pom.xml

```diff
@@ -92,6 +92,11 @@
       <version>${fasterxml.jackson.version}</version>
     </dependency>
     <dependency>
+      <groupId>com.ibm.icu</groupId>
```

Comment:
OK, yes that's a cat-A license so it's OK. With any dependency I'd also want to check whether it brings in anything else under a different license or whether it's particularly large, etc.

Disregard my other comment. I think I was thinking of the fact that Lucene already uses this.